.gitignore

```diff
@@ -117,4 +117,8 @@ fastlane/test_output
 
 iOSInjectionProject/
 
+.DS_Store
```

Comment:
This is not working since I see the file is pushed to GitHub... make sure you have the correct path in the .gitignore file.